test/expression_test.rb

```diff
@@ -58,4 +58,14 @@ def test_missing_receiver
     assert_equal nil, expr.evaluate("user" => user)
     assert_equal nil, expr.evaluate({})
   end
+
+  def test_missing_collection_receiver
```

Comment:
@senny you said on [#52](https://github.com/senny/sablon/pull/52) that's reasonable to raise error when the collection does not exist. I can revert this commit because it's on my working branch, however I'd like to ask you to consider again since it's very painful to the end user to put a lot of guards on the template. Thanks!